telegram/_files/inputmedia.py

```diff
@@ -414,6 +485,7 @@ def __init__(
             self.height: Optional[int] = height
             self.duration: Optional[int] = duration
             self.thumb: Optional[Union[str, InputFile]] = self._parse_thumb_input(thumb)
+            self.thumbnail: Optional[Union[str, InputFile]] = self._parse_thumb_input(thumbnail)
```

Comment:
```suggestion
            self.thumbnail: Optional[Union[str, InputFile]] = self.thumb
```

similar in other places :)

Reply:
Hm... I will save this for a bit later, too tired to understand this now, I guess :) I tried the quick change and it failed the tests